pep-0394.txt

```diff
@@ -141,6 +140,19 @@ For developers
   a container or virtual environment) may continue to use the ``python``
   command name.
 
+For end users of Python
+-----------------------
+
+* While far from being universally available, ``python`` remains the
+  preferred spelling for explicitly invoking Python, as this is the
+  spelling that virtual environments make consistently available
+  across different platforms and Python installations.
+* For software that is not distributed with (or developed for) your system,
+  we recommend using a virtual environment, possibly with an environment
+  manager like ``conda`` or ``pipenv``, to help avoid disrupting your system
```

Comment:
Should we mention the built-in `venv`?

Reply:
Perhaps in parentheses in the first bullet point? "... that virtual environments (as created by the standard library's `venv` module or the PyPA's `virtualenv` project) consistently ..."

Reply:
I'm going to leave this for now, as these do get mentioned further up.